src/App.tsx

```diff
@@ -1,19 +1,76 @@
-import React, { Suspense } from "react";
+import React, { Suspense, useCallback } from "react";
 import { ThemeProvider } from "styled-components";
 import { BrowserRouter as Router, Route, Switch } from "react-router-dom";
 import Home from "./components/Home";
 import Projects from "./components/Projects";
 import theme from "./components/theme";
-import NavMenu from "./components/NavMenu";
+import styled from "styled-components";
+import i18next from "i18next";
 import "./css/reset.css";
+import {
+  color,
+  ColorProps,
+  space,
+  SpaceProps,
+  typography,
+  TypographyProps,
+  layout,
+  LayoutProps,
+  position,
+  PositionProps,
+} from "styled-system";
+
+type LanguageButtonProps = SpaceProps &
+  ColorProps &
+  TypographyProps &
+  PositionProps &
+  LayoutProps;
+
+const Container = styled.div<LanguageButtonProps>`
+  ${position}
+  ${layout}
+  cursor: pointer;
+`;
+
+const LangButton = styled.p<LanguageButtonProps>`
+  ${typography}
+  ${color}
+  ${space}
+`;
 
 const App = () => {
+  const onLangClick = useCallback(
+    (countryId: string) => () => i18next.changeLanguage(countryId),
+    []
+  );
   return (
     <Router>
       <ThemeProvider theme={theme}>
         <div className="App">
           <Suspense fallback={<div>Loading</div>}>
-            <NavMenu />
+            <Container
+              position="absolute"
+              right={[75, 75, 75, 100, 100]}
+              top={28}
```

Comment:
Feel like right and top should be `30` here to be consistent with the content in the other four corners?
![image](https://user-images.githubusercontent.com/12934669/86607706-85951000-bfa1-11ea-820c-c1c3ffb7bf81.png)

Reply:
This was to leave space for the menu button once merged into the `develop` branch! I feel you on the `top` prop though.